Library/Homebrew/cmd/audit.rb

```diff
@@ -152,11 +142,21 @@ def initialize(formula, options = {})
     @formula = formula
     @strict = !!options[:strict]
     @online = !!options[:online]
+    # Accept precomputed style offense results, for efficiency
+    @style_offenses = options[:style_offenses]
     @problems = []
     @text = FormulaText.new(formula.path)
     @specs = %w[stable devel head].map { |s| formula.send(s) }.compact
   end
 
+  def audit_style
+    return unless @style_offenses
+    display_cop_names = ARGV.include?("--display-cop-names")
+    @style_offenses.each do
+      |offense| problem offense.to_s(:display_cop_name => display_cop_names)
```

Comment:
`|offense|` should be at the end of the preceding line, not at the start of this line.